static/src/javascripts/projects/commercial/modules/third-party-tags/remarketing.js

```diff
@@ -9,8 +13,13 @@ const onLoad = () => {
     });
 };
 
-export const remarketing: ThirdPartyTag = {
-    shouldRun: config.get('switches.remarketing'),
-    url: '//www.googleadservices.com/pagead/conversion_async.js',
-    onLoad,
+export const remarketing: () => ThirdPartyTag = () => {
+    const consent = getAdConsentState(thirdPartyTrackingAdConsent);
+
+    return {
+        shouldRun:
+            config.get('switches.remarketing') && (consent || consent == null),
```

Comment:
According to the ticket `null == no`, so `shouldRun` should be false if `consent == null`

Reply:
👍